### DISCUSSION TITLE
Import Geopoint data

### DISCUSSION THREAD

**Discussion by NGProject** [AntlerVC/firetable#791] [Support / Q&A] [answered]
Hi! I need to import a csv file which contains gps positions. 
How should i format geopoint cells? I only get invalids data...

**Comment by BeeBombshell** [comment, marked as answer]
Hey @NGProject, I would suggest exporting the table as `.csv` so that you get familiar with the format in which the coordinates have been stored. I created a small video to demonstrate the process after you have exported the table and cleared the existing field values (to avoid duplicates).

https://user-images.githubusercontent.com/53828745/201122639-5a891178-157a-4e7d-b8aa-61ac7daec51e.mp4

As you can see, I added a negative sign(-) for the longitude value. This is because, as per standards, the latitude value is taken positive starting from **North** and the longitude value is taken positive starting from **West**. So, whenever we're dealing with latitudes starting from **South** and longitudes starting from **East**, we'll have to prefix the value with a negative sign.
